schemas/system_profile/v1.yaml

```
@@ -780,6 +780,28 @@ $defs:
             type: boolean
             x-indexed: false
             example: true
+      rhel_ai:
+        description: Object containing information about RHEL AI
+        type: object
+        properties:
+          variant:
+            description: RHEL AI VARIANT
+            type: string
+            example: "RHEL AI"
+            maxLength: 16
+          rhel_ai_version_id:
+            description: RHEL AI VERSION ID
+            type: string
+            example: "v1.1.3"
+            maxLength: 16
+          nvidia_gpu_models:
+            description: Model name of Nvidia GPUs in the GPU index order
+            type: array
+            items:
+              description: The Nvidia GPU model
+              type: string
+              example: "NVIDIA T1000, Tesla V100-PCIE-16GB"
```

If this is meant to be one value in the array, I think it would be better to use one or the other. Doesn't affect the functionality but may cause confusion
```suggestion
              example: "Tesla V100-PCIE-16GB"
```